### ISSUE TITLE
Fit image to screen size on Tizen (Samsung TV)

### ISSUE THREAD

**Issue by POLFLAPS** [Stremio/stremio-features#959]
**Stremio Version**
Tizen (Samsung TV)

**Is your feature request related to a problem? Please describe.**
Video formats (height and width) usually do not match the aspect ratio of my Smart TV screen. This causes "letterboxing" where black (empty) space is added around either the top/bottom or left/right sides of the image. It also causes my screen not to be used to its fullest capacity. This is annoying and less immersive.

**Describe the solution you'd like**
On the Chromecast app, when playing a video stream I can go to the playback settings, and choose between a few different screen resizing options (Fit to screen, automatic, ...). I would like to see this functionality in the Samsung TV app too.

**Describe alternatives you've considered**
The Tizen TV settings do not allow any options for this setting to be tweaked for apps.

**Additional context**

![image](https://github.com/user-attachments/assets/6d1defe6-a530-4963-a9b3-aea8dc06a654)

**Comment by claufecam** [comment]
It would be amazing to have this feature on Samsung TV's Stremio. It's a real shame not having the option to remove the black bars from movies like on Android. It's even sad to watch a movie like that 😔

**Comment by rja120** [comment]
Yes, I totaly agree it would be nice to have the option to adjust aspect ratio on Tizen
